www/.eleventy.js

```diff
@@ -1,9 +1,12 @@
-const { DateTime } = require('luxon');
-const UglifyJS = require('uglify-es');
-const htmlmin = require('html-minifier');
-const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
-
-module.exports = function (eleventyConfig) {
+import eleventyNavigationPlugin from '@11ty/eleventy-navigation';
```

Comment:
New version of Eleventy allows ESM, so use that instead. No other significant config changes.